src/js/overlay.js

```diff
@@ -159,6 +163,10 @@ Overlay.prototype.render = function() {
 	const wrapperEl = document.createElement('div');
 	wrapperEl.className = 'o-overlay';
 	wrapperEl.classList.add('o-overlay--' + this.id.replace(' ', '-'));
+	wrapperEl.setAttribute('role', 'dialog');
+	if(this.opts.heading) {
+		wrapperEl.setAttribute('aria-labelledby', this.opts.heading.title);
```

Comment:
Yep, you're correct. After lunch I'll check what's best to be used in this situation and update it.